test/fixtures/test/widgets.json

```diff
@@ -0,0 +1,41 @@
+[
+  {
+    "widget_type": "dob",
+    "name": "dob",
+    "definition": {
+      "display_name": "Date of Birth"
+    }
+  },
+  {
+    "widget_type": "sex",
+    "name": "sex",
+    "definition": {
+      "display_name": "Sex"
+    }
+  },
+  {
+    "widget_type": "status",
+    "name": "status",
+    "definition": {
+      "display_name": "Status"
+    }
+  },
+  {
+    "widget_type": "divider",
+    "name": "divider"
+  },
```

Comment:
Not a _huge_ deal since this is a test fixture, but `definition` is defined as NOT NULL. In this case it would be `"definition": {}`

I also specify it is an object in our OpenAPI spec, but this is actually incorrect, it should allow any non-null value. I will fix this

Reply:
:+1: should be ok.  Divider won't attempt to use that attribute